rpki-validator/src/main/java/net/ripe/rpki/validator3/jobs/JobExecutor.java

```diff
@@ -167,14 +167,14 @@ public void sequence(Job... newJobs) {
         private Job job;
     }
 
-    @EqualsAndHashCode
+    @EqualsAndHashCode(callSuper = true)
```

Comment:
Oh! JobExecutor should be deleted at all, but somehow it leaked into the master branch, so don't bother changing it.